pytest_jira.py

```diff
@@ -15,13 +15,20 @@
 from jira.client import JIRA
 
 
+def console_write(message, *args, **kwargs):
+    tr = pytest.config.pluginmanager.getplugin('terminalreporter')
```

Comment:
AttributeError: 'module' object has no attribute 'config'
We can not access pytest.config from here.